src/intervals/functions.jl

```diff
@@ -266,3 +266,22 @@ for f in (:log, :log2, :log10, :log1p)
 
         end
 end
+
+for f in (:erf, :erfc)
+    @eval function($f)(x::BigFloat, r::RoundingMode)
+        setrounding(BigFloat, r) do
+            ($f)(x)
+        end
+    end
+
```

Comment:
For the `Float64` case is `@eval ($f)(a::Interval{Float64}) = atomic(Interval{Float64}, $f(big53(a)))` ok? (I saw this for the `exp2` function, but it is not clear to me what `big53` is)

Reply:
`big53` is defined in the package. It converts the interval to a BigFloat interval with 53 bits of precision (i.e. the same as Interval{Float64}). You don't need `atomic`, just `convert`.